packages/rspack/tests/configCases/source-map/cheap/index.js

```diff
@@ -5,7 +5,7 @@ require("./index.scss");
 it("should only map transformed lines if cheap options is used", async () => {
 	const source = fs.readFileSync(__dirname + "/main.css.map", "utf-8");
 	const map = JSON.parse(source);
-	expect(map.sources).toContain("./index.scss");
+	expect(map.sources.some(s => s.includes("./index.scss"))).toBeTruthy();
```

Comment:
The item of `sources` changes from `"./index.scss"` to `"../../../../node_modules/sass-loader/dist/cjs.js!./index.scss"` and I personally think this is working as intended to mark the processed loaders. Any comments? @ahabhgk

Reply:
lgtm